lib/server/routes/health.js

```diff
@@ -23,7 +27,7 @@ HealthRouter.prototype.health = function(req, res) {
 
 HealthRouter.prototype._definitions = function() {
   return [
-    ['GET', '/health', this.health]
+    ['GET', '/health', this.getLimiter(limiter(1000)), this.health]
```

Comment:
We may not need to have limiter here, since this is used internally may not be desirable an it doesn't do much.